ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -606,12 +610,14 @@ export default class ConfirmTransactionBase extends Component {
                 detailText={renderTotalDetailText()}
                 detailTotal={renderTotalDetailTotal()}
                 subTitle={t('transactionDetailGasTotalSubtitle')}
-                subText={t('editGasSubTextAmount', [
-                  <b key="editGasSubTextAmountLabel">
-                    {t('editGasSubTextAmountLabel')}
-                  </b>,
-                  renderTotalMaxAmount(),
-                ])}
+                subText={
+                  <>
+                    <b key="editGasSubTextAmountLabel">
+                      {t('editGasSubTextAmountLabel')}
+                    </b>
```

Comment:
Would `<strong>` tags be more semantically correct here?

Reply:
fixed